lambda_layers/core_layer/python/core_layer/handler/notification_template_handler.py

```diff
@@ -37,21 +38,21 @@ def get_notification_template(self, notification_type: str, message_type: str, c
         return self._get_notification_template(notification_type, message_type, content_type, language)
 
 
-class S3NotificationTemplateHandler(NotificationTemplateHandlerBase):
+class NotificationTemplateHandler(NotificationTemplateHandlerBase):
     def _get_notification_template(self, notification_type: str, message_type: str, content_type: str, language: str):
-        client = BotoClientProvider().get_client("s3")
 
-        response = client.get_object(
-            # 'codetekt-notification-templates-dev',
-            Bucket=os.environ['NOTIFICATION_TEMPLATE_BUCKET'],
-            Key=f'{message_type}/{language}/{notification_type}/{content_type}.txt'
-        )
+        template_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 
+                                                          '../../../../..',
+                                                          'lambda_functions/notification_service/resources/mail_templates', 
+                                                          f'{message_type}/{language}/{notification_type}/{content_type}.txt'
```

Comment:
Findet sich hier nicht evtl. eine einfachere Möglichkeit die Dateien abzulegen? Ohne dass man einen so umständlichen Pfad zusammenbasteln muss?

Reply:
hast Du einen Vorschlag?

Reply:
Du könntest den NotificationTempalteHandler aus dem Layer in den eigentlichen Funktionscode verschieben.
Der wird ja aus keinem anderen Lambda Service aufgerufen als dem notification_service, oder?

Wir können es aber auch erstmal so ausprobieren. Ich bin mir ohnehin nicht sicher, ob die Verzeichnisstruktur der deployten Lambda Funktion genau so ist wie im git Repo.

Reply:
Dann lass es uns erstmal so probieren :)